src/commands/admin.ts

```diff
@@ -0,0 +1,22 @@
+import { Message } from "discord.js"
+
+const { Command } = require('discord-akairo')
+
+const pluralize = require('pluralize')
+
+class AdminCommand extends Command {
+    constructor() {
+        super('admin', {
+            aliases: ['admin']
+        })
+    }
+
+    exec(message: Message) {
+        if (message.author.id === '139468879008235520') {
```

Comment:
lol

Reply:
this is from $admin, i just force pushed it through last night so I could see it. I guess I could make that an envvar?

Reply:
I resolved this in a662827